src/components/pages/Registration/RegisterFormComponents/StudentForm.js

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from 'react';
+import '../../../../styles/registration.less';
+import avi1 from '../../../../img/Avatars/Avatar01.svg';
+import avi2 from '../../../../img/Avatars/Avatar03.svg';
+import avi3 from '../../../../img/Avatars/Avatar15.svg';
+import avi4 from '../../../../img/Avatars/Avatar10.svg';
+import avi5 from '../../../../img/Avatars/Avatar06.svg';
+import avi6 from '../../../../img/Avatars/Avatar07.svg';
+import avi7 from '../../../../img/Avatars/Avatar16.svg';
```

Comment:
Great job implementing the form and associated routes! One thing that caught my eye is the redundancy of importing each avatar individually and having multiple `<img` tags for each avatar. 

Consider creating an image component and storing all the avatars in an array and mapping through them for your desired avatar.  Great job regardless!

Reply:
I agree! this was done just to get it to work, but can definitely be abstracted using an array of images.  I'll add this to future updates.